src/app/MessageProvider.tsx

```diff
@@ -17,8 +17,17 @@ async function polyfill(locale: 'en') {
 
 polyfill('en');
 
+const cache = createIntlCache();
+
+export const intl = createIntl({
+  locale: 'en',
+  messages: enMessages,
+}, cache);
+
 // tslint:disable-next-line:variable-name
-const MessageProvider: React.SFC<{onError?: () => void}> = (props) =>
-  <IntlProvider onError={props.onError} locale='en' messages={enMessages}>{props.children}</IntlProvider>;
+const MessageProvider: React.FC<{}> = (props) =>
+  <RawIntlProvider value={intl}>
```

Comment:
@TomWoodward Importing the useServices() hook here instead created hook usage errors in some tests =/